packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py

```diff
@@ -74,6 +74,11 @@ class SpanAttributes:
     LLM_OPENAI_API_VERSION = "gen_ai.openai.api_version"
     LLM_OPENAI_API_TYPE = "gen_ai.openai.api_type"
 
+    # Anthropic
```

Comment:
I am afraid that this PR won't pass tests, because `pyproject.toml` in the `anthropic` package looks at `0.4.1`, basically the PyPI version, but my changes rely on the new attributes here. Let me know if I need to merge this first somehow

Reply:
Yeah I need to manually publish the semconv package cause poetry doesn't support deeply nested local dependencies :/ 
https://github.com/python-poetry/poetry/issues/2270